ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -157,6 +157,12 @@ do
   yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" "$f"
 done
 
+# workaround for bug https://issues.redhat.com/browse/OCPBUGS-33921
+if [ -f ${SHARED_DIR}/external_lb_patch_install_config.yaml ]; then
+  elbip=$(yq ".api_vip" "${SHARED_DIR}/vips.yaml")
```

Comment:
Hi can we add this directly in the patch step?

Reply:
do you mean update baremetal-lab-ipi-conf-external-lb-commands.sh with 
```
cat > "${SHARED_DIR}/external_lb_patch_install_config.yaml" <<'EOF'
networking:
  machineNetwork:
  - cidr: ${INTERNAL_NET_CIDR}
  - cidr: $(yq ".api_vip" "${SHARED_DIR}/external_vips.yaml")/32
platform:
  baremetal:
    loadBalancer:
      type: UserManaged
EOF
```

Reply:
Yes, just remove the single quote ' around EOF to enable variable substitution

Reply:
sounds like `INTERNAL_NET_CIDR` is not valid here yet. 

INTERNAL_NET_CIDR: unbound variable

Reply:
Add it to the variables of the step as we do for the others

Reply:
Yeah, looks good now.